generators/server/templates/src/test/java/package/web/rest/_LogsResourceIntTest.java

```diff
@@ -19,10 +19,11 @@
 package <%= packageName %>.web.rest;
 
 <%_ if (databaseType === 'cassandra') { _%>
-import <%= packageName %>.AbstractCassandraTest;
+import <%=packageName%>.AbstractCassandraTest;
```

Comment:
Should we have a rule for spacing in template ? Our current templates have a mix.